python/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "lancedb-python"
-version = "0.16.0-beta.0"
+version = "0.16.0-beta.1"
```

Comment:
You do not need to bump the version. That happens automatically in release process.

Reply:
No, i don't need - i did not find the rules, so i thought the minor versions are raised every time. I'm reverting back this change.

Reply:
@wjones127 Can you please check the PR again?